src/Operation/Watch.php

```diff
@@ -264,7 +264,7 @@ public function __construct(Manager $manager, $databaseName, $collectionName, ar
 
         $this->manager = $manager;
         $this->databaseName = (string) $databaseName;
```

Comment:
I don't think this `(string)` cast is necessary given the `if ($databaseName === null)` conditional above that sets it to "admin" for cluster-wide change streams. Feel free to remove.

Reply:
You are correct - for some reason the PhpStorm tooling still saw the cast as required and didn't remove it automatically. I removed this.